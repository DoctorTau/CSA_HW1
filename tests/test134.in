90
192 -3411 -6522 -8853 -7339 -232 -7165 -1073 141 -8239 -7332 8999 -318 -5047 -3246 7807 4457 4420 1326 -5254 -4746 -2646 1650 -6452 -9130 -9040 491 857 -9244 5607 6200 6615 6873 -2389 -5369 1350 6595 2874 -6619 2369 -140 802 -5619 9045 4018 -4351 -7956 -1962 -6705 1275 24 5171 6981 8021 3901 -7485 -5493 5596 -5146 736 -9466 -210 -243 -5440 -6039 1896 -6602 202 1797 -7803 -2292 -1608 2430 6905 9519 8516 9989 6143 7621 3570 7350 3237 9226 -4951 8403 -7934 7608 4970 -1991 535 