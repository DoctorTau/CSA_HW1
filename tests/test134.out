-6522 -232 -7332 -318 -3246 -5254 -4746 -2646 -6452 -9130 -9040 -9244 -140 -7956 -1962 -5146 -9466 -210 -5440 -6602 -2292 -1608 -7934 